scripts/rocal_training_setup.py

```diff
@@ -124,7 +124,8 @@ def prefetched_loader(self, loader, rocal_cpu):
 
         stream = torch.cuda.Stream()
         first = True
-
+        input = None 

```

Comment:
@hansely this is my change but as you are working on this PR, can you get rid of lines 127 and 128, we have to resolve it later